guides/doc-Monitoring_Project/topics/proc_configuring-pcp-data-collection.adoc

```diff
@@ -0,0 +1,109 @@
+[id='configuring-pcp-data-collection_{context}']
+= Configuring PCP Data Collection
+
+This procedure describes how to configure PCP to collect metrics about processes, {Project}, Apache HTTP Server, and PostgreSQL.
+
+.Procedure
+. To configure PCP to collect data about {Project} processes, create the `/var/lib/pcp/pmdas/proc/hotproc.conf` file and include the following content:
++
+----
+#pmdahotproc
+Version 1.0
+
+(
+  fname == "java" ||
+  fname == "redis-server" ||
+  (fname == "postgres" && psargs ~ /-D/) ||
+  psargs ~ /puma/ ||
+ifndef::foreman-el,foreman-deb[]
+  psargs ~ /pulpcore.app.wsgi:application/ ||
+  psargs ~ /pulpcore.content:server/ ||
+  psargs ~ /pulpcore-worker/ ||
+endif::[]
+  psargs ~ /sidekiq/ ||
+  psargs ~ /smart-proxy/
+)
+----
++
+By default, PCP collects basic system metrics.
+This step enables detailed metrics about the following {Project} processes:
++
+* Java
+* PostgreSQL
+* Redis
+* Dynflow
+* Puma
+ifndef::foreman-el,foreman-deb[]
+* Pulpcore
+endif::[]
+
+. Configure PCP to log the process metrics being collected.
++
+----
+# mkdir -p /var/lib/pcp/config/pmlogconf/foreman-hotproc
+# cat >/var/lib/pcp/config/pmlogconf/foreman-hotproc/summary << EOF
+#pmlogconf-setup 2.0
```

Comment:
```suggestion
# pmlogconf-setup 2.0
```

Reply:
no, that's how a pmlogconf file has to look like, no space